README.md

```diff
@@ -46,31 +46,18 @@ $ git clone https://github.com/j-marple-dev/model_compression.git
 $ cd model_compression
 ```
 
-1. Install PyTorch 1.5.1 and Torchvision 0.6.1 (See the following official instruction).
+1. Create virtual environment
 ```bash
-$ conda install pytorch==1.5.1 torchvision==0.6.1 -c pytorch
-```
-
-2. Install `progressbar2`
-
-```bash
-$ conda install -c conda-forge progressbar2
-```
-
-###### For users
-Install packages required to execute the code. Just type:
-```bash
-$ make dep
+$ make dev
 ```
 
-###### For developers
-
-If you want to modify code you should configure formatting and linting settings. It automatically runs formatting and linting when you commit the code. Just type:
```

Comment:
Since environment.yml installs all the libraries at once, I suggest removing the users/developers installation procedure.